.github/ISSUE_TEMPLATE/bug_template.yml

```diff
@@ -0,0 +1,72 @@
+name: "Bug report"
+description: Report a bug for this project.
+labels: ["bug", "🔍 Ready for Review"]
+assignees:
+- peletron
+body:
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+      - label: I have searched the existing issues
+        required: true
+  - type: input
+    id: package-version
+    attributes:
+      label: Package version
+      description: What version of go-ipfs-log package are you using?
+      placeholder: v1.8.0
+    validations:
+      required: true
+  - type: input
+    id: go-version
+    attributes:
+      label: Golang version
+      description: What version of Golang are you using?
+      placeholder: go1.18.4
+    validations:
+      required: true
+  - type: textarea
+    id: use-case
+    attributes:
+      label: Use case
+      description: Provide a detailed explanation of your use case.
+    validations:
+      required: true
+  - type: textarea
+    id: code-snippet
+    attributes:
+      label: Code snippet
+      description: Provide a code snippet where you are getting the issue.
+    validations:
+      required: true
```

Comment:
Can you merge this two sections please?
Into something like:
```
label: Bug description
description: Provide a bug description and a code snippet if applicable.
```

Reply:
I merged two sections!